src/sonic-config-engine/config_samples.py

```diff
@@ -10,6 +10,16 @@
 else:
     UNICODE_TYPE = unicode
 
+def generate_common_config(data):
+    data['FLEX_COUNTER_TABLE'] = {
+        'ACL': {
+            'FLEX_COUNTER_STATUS': 'disable',
+            'FLEX_COUNTER_DELAY_STATUS': 'true',
+            'POLL_INTERVAL': '10000'
```

Comment:
@stepanblyschak Could you please help? What could be the possible POLL_INTERVAL range?